client/src/pages/Home.js

```diff
@@ -33,7 +33,7 @@ export function Home() {
 					alt="Just the React logo"
 				/>
 				<h1 className="message" data-qa="message">
-					{message}
+					{message}+"Hello testing"
```

Comment:
@ali1373107  can you replace the quote marks with \&quot;  the quotes are causing issues with eslint, but the code works. 

i can merge after code looks like this  {message}+\&quot;Hello testing\&quot;

Reply:
A better approach would be using string literal:

```
${`${message} Hello testing`}
```